src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -1,12 +1,7 @@
 package core.mate.academy.model;
 
-/**
- * Add some custom fields that could be only in Bulldozer
- * Do not remove no-args constructor
- */
 public class Bulldozer extends Machine {
-    public Bulldozer() {
-    }
+    private int size = 0;
 
     @Override
     public void doWork() {
```

Comment:
/**
 * Add some custom fields that could be only in Bulldozer
 * Do not remove no-args constructor
 */

Reply:
![image](https://user-images.githubusercontent.com/79330731/195575190-dd889df3-7a0a-45cc-873c-cd5435a13edc.png)